deploy/charts/external-secrets/values.yaml

```diff
@@ -244,6 +255,12 @@ certController:
     # -- Pod priority class name.
   priorityClassName: ""
 
+  # -- Pod disruption budget - for more details see https://kubernetes.io/docs/concepts/workloads/pods/disruptions/
+  podDisruptionBudget:
```

Comment:
certController doesn't have a way to set the replicaCount. without that it doesn't make sense to add a pdb for it.
Can you add `certController.replicaCount`?

Reply:
it makes sense - done

Reply:
done